cmd/notary/main.go

```diff
@@ -1,96 +1,71 @@
 package main
 
 import (
-	"crypto/x509"
 	"fmt"
 	"os"
 	"os/user"
-	"path"
-	"time"
+	"path/filepath"
 
 	"github.com/Sirupsen/logrus"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 
-	"github.com/docker/notary/trustmanager"
+	"github.com/docker/notary/pkg/passphrase"
 )
 
 const configFileName string = "config"
-const configPath string = ".docker/trust/"
-const trustDir string = "trusted_certificates/"
-const privDir string = "private/"
-const rootKeysDir string = "root_keys/"
+const defaultTrustDir string = ".notary/"
+const defaultServerURL = "https://notary-server:4443"
```

Comment:
We're thinking about adding a default server at thenotary.io that people can use as the default trust server. Unfortunately setting that up (and how we would enforce GUN ownership), is currently an unsolved issue.